.changesets/breaking_drop_reuse_fragment.md

```diff
@@ -0,0 +1,7 @@
+### Drop experimental reuse fragment query optimization option ([PR #6354](https://github.com/apollographql/router/pull/6354))
+
+Drop support for the experimental reuse fragment query optimization. This implementation was not only very slow but also very buggy due to its complexity.
+
+Auto generation of fragments is a much simpler (and faster) algorithm that in most cases produces better results. Fragment auto generation is the default optimization since v1.58 release.
```

Comment:
```suggestion
### Drop experimental reuse fragment query optimization option ([PR #6354](https://github.com/apollographql/router/pull/6354))

We've dropped support for the `experimental_reuse_query_fragments` option. Its implementation didn't achieve good enough performance or reliability. 

In its place, the router uses fragment auto generation. As the default implementation since router v1.58, in most cases fragment auto generation achieves better performance and results.
```

Reply:
Sorry the prep already happened.